Expensify/App
# fix: Change emoji navigation with arrow keys

### Details

- Allow the user to go to the first emoji on click of the right arrow
- Allow the user to go to the search input clicking on the left arrow while first emoji is selected.

### Fixed Issues

$ https://github.com/Expensify/App/issues/7035

### Tests
1. Tested `ArrowRight` to ensure that it selects the first emoji when the cursor is at the end of the string
2. `ArrowDown` should ensure that it goes to the first emoji irrespective of the cursor position
3. `ArrowUp` and `ArrowLeft` would select the text in the input and rest of the cursor works fine.



### QA Steps
1. Open any chat
2. Open the emoji picker and type a search term
3. If the cursor is at the end of the input, and click the `right arrow` key
4. It should select the first emoji
5. When the first emoji is selected, and you click `left arrow`, it should focus the search input and select the text




### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/3069065/149575845-efb0bb4b-148f-4b3a-a5b6-e031c62dda49.mov


#### Mobile Web

NA

#### Desktop

https://user-images.githubusercontent.com/3069065/149575877-56a5625c-0b75-4756-899f-9eae40e87856.mov

#### iOS

NA

#### Android

NA
